lib/web_ui/lib/src/ui/lerp.dart

```diff
@@ -2,29 +2,29 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-// @dart = 2.6
+// @dart = 2.9
 part of ui;
 
 /// Linearly interpolate between two numbers.
-double/*?*/ lerpDouble(num/*?*/ a, num/*?*/ b, double/*!*/ t) {
+double? lerpDouble(num? a, num? b, double t) {
   if (a == null && b == null) {
     return null;
   }
   a ??= 0.0;
   b ??= 0.0;
-  return a + (b - a) * t;
+  return a + (b - a) * t as double?;
```

Comment:
just `as double`.  The result can't be null.

Reply:
@stereotype441 good feedback for the tool?

Reply:
Done.